app/screens/TodoList/components/TodoItem/layout.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import { View } from 'react-native';
+import { View, TouchableOpacity, Text, StyleSheet } from 'react-native';
 
 import CheckBox from './components/CheckBox';
 
-function TodoItem({ onCheck, onDelete, isChecked }) {
+function TodoItem({ onCheck, onDelete, isChecked, item }) {
```

Comment:
Please check that Styles should come from styles file and not from the same layout file.

The style is used by index and layout files if it needed.